### DISCUSSION TITLE
utf8 win console question

### DISCUSSION THREAD

**Discussion by k1attila1** [edubart/nelua-lang#158] [General]
Hi
i try this : 

print 'ő'

in Win console i get : ┼Ĺ   (hex : c5,91)

how can i solve it in nelua without chcp 65001 ?

thank you

**Comment by codehz** [comment]
Maybe attach an utf8 manifest would help.
See 
https://docs.microsoft.com/en-us/windows/apps/design/globalizing/use-utf8-code-page

**Comment by ghost** [comment]
> how can i solve it in nelua without chcp 65001 ?

That's a problem with the windows console(command prompt)

One of the possible solution is to use a terminal which [mintty](https://mintty.github.io/)  internally like "git bash", "cygwin's terminal", "msys2's terminal", "MSYS's terminal", "WSL"
